src/main/java/mate/academy/spring/boot/model/Book.java

```diff
@@ -6,16 +6,16 @@
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
 import jakarta.persistence.Table;
-import lombok.Getter;
-import lombok.Setter;
-import lombok.ToString;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 import java.math.BigDecimal;
 
+@Data
 @Entity
-@Getter
```

Comment:
Please revert `@Getter, @Setter and @ToString` annotations, try to avoid using `@Data` for entities

Reply:
Please, fix the comment.